web/modules/custom/dpl_event/src/EventSubscriber/EventsRedirect.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Drupal\dpl_event\EventSubscriber;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\recurring_events\Entity\EventInstance;
+use Drupal\recurring_events\Entity\EventSeries;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Redirects requests for eventseries entities.
+ *
+ * @package Drupal\dpl_event\EventSubscriber
+ */
+class EventsRedirect implements EventSubscriberInterface {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Constructs a new EventsRedirect object.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
+    $this->entityTypeManager = $entity_type_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents(): array {
+    return [
+      KernelEvents::REQUEST => [
+        ['checkEventSeriesRedirect'],
+        ['checkEditInstanceRedirect'],
+      ],
+    ];
```

Comment:
Please reconsider this.

On the PR environment, I cannot tell that viewing the single instance series, https://varnish.pr-1049.dpl-cms.dplplat01.dpl.reload.dk/bronshoj-bibliotek/events/design-teknologi/fernisering-moderne-dans-0, redirects to the instance, https://varnish.pr-1049.dpl-cms.dplplat01.dpl.reload.dk/bronshoj-bibliotek/events/design-teknologi/fernisering-moderne-dans.

On the other hand the redirect from the instance, https://varnish.pr-1049.dpl-cms.dplplat01.dpl.reload.dk/bronshoj-bibliotek/events/design-teknologi/fernisering-moderne-dans to the edit form for the series work, https://varnish.pr-1049.dpl-cms.dplplat01.dpl.reload.dk/events/series/3/edit .

If you cannot determine why this occurs then consider splitting this into two different event subscribers.